app/controllers/subsidy/detail/step/detail.js

```diff
@@ -107,187 +98,8 @@ export default class SubsidyDetailStepDetailController extends Controller {
     return this.model.step.get('deadline').content;
   }
 
-  // TODO what is this?
-  @action
-  registerObserver() {
-    this.formStore.registerObserver(() => {
-      this.setTriplesForTables();
-    });
-  }
```

Comment:
are you sure we dont need this? I am not sure either so just checking

Reply:
I'm not sure I couldn't see it get called once so I figured it wasn't needed anymore, but maybe keep it to be safe?

Reply:
yeah, its because if it is used, we might end up looking hours/days debugging not knowing why something doesnt work as expected. I'll be adding it back, dont worry :)